spec/commands/feed/fetch_one_spec.rb

```diff
@@ -1,94 +1,82 @@
 # frozen_string_literal: true
 
 RSpec.describe Feed::FetchOne do
-  let(:daring_fireball) do
-    double(
-      id: 1,
-      url: "http://daringfireball.com/feed",
-      last_fetched: Time.zone.local(2013, 1, 1),
-      stories: []
-    )
+  def create_entry(opts: {})
+    entry = {
+      comments: "https://news.ycombinator.com/item?id=38246668",
+      published: Time.zone.now,
+      summary: "<a href=\"https://news.ycombinator.com/item?id=38246682\">Comments</a>",
+      title: "Run LLMs on my own Mac fast and efficient",
+      url: "https://www.secondstate.io/articles/fast-llm-inference",
+      content: "",
+      id: "test"
+    }.merge(opts)
+    double(entry)
   end
 
-  before { stub_request(:get, "http://daringfireball.com/feed") }
+  def create_raw_feed(last_modified: Time.zone.now, entries: [])
+    double(last_modified:, entries:)
```

Comment:
One possibility might be to make this method do the stubbing as well:

```suggestion
  def stub_raw_feed(last_modified: Time.zone.now, entries: [])
    raw_feed = double(last_modified:, entries:)
    allow(described_class).to receive(:fetch_raw_feed).and_return(raw_feed)
```